imap/lmtpd.c

```diff
@@ -883,11 +893,25 @@ int deliver(message_data_t *msgdata, char *authuser,
             strarray_t flags = STRARRAY_INITIALIZER;
             struct imap4flags imap4flags = { &flags, authstate };
 
+            const char *userid = mbname_userid(mbname);
+            struct proc_limits limits;
+            limits.servicename = config_ident;
+            limits.clienthost = lmtpd_clienthost;
+            limits.userid = userid;
+            if (proc_checklimits(&limits)) {
+                if (limits.maxhost && limits.maxhost <= limits.host) r = IMAP_LIMIT_HOST;
+                else if (limts.maxuser && limits.maxuser <= limits.user) r = IMAP_LIMIT_USER;
```

Comment:
`limts` typo... why isn't this warning??

Reply:
Oh it's a compile error, good, and that's why CI is failing:

```
imap/lmtpd.c: In function 'deliver':
imap/lmtpd.c:903:26: error: 'limts' undeclared (first use in this function); did you mean 'limits'?
                 else if (limts.maxuser && limits.maxuser <= limits.user) r = IMAP_LIMIT_USER;
                          ^~~~~
                          limits
imap/lmtpd.c:903:26: note: each undeclared identifier is reported only once for each function it appears in
  CC       imap/lmtpd-sync_support.o
```

I was worried we might have some stray `limts` symbol somewhere that was being used instead

Reply:
yeah, just a typo -fixed and pushed!